Sources/Core/Sources/ECS/Singles/InputState.swift

```diff
@@ -33,6 +33,11 @@ public final class InputState: SingleComponent {
   /// The time since jump last changed from not pressed to pressed.
   public private(set) var ticksSinceJumpPressed: Int = 0
 
+  /// Whether sprint is currently toggled. Only used if toggle sprint is enabled
```

Comment:
Tiny nitpick thing, but I generally try to end doc comments with a full stop for consistency